tailcfg/tailcfg.go

```diff
@@ -175,6 +175,21 @@ type UserProfile struct {
 	// Roles exists for legacy reasons, to keep old macOS clients
 	// happy. It JSON marshals as [].
 	Roles emptyStructJSONSlice
+
+	// Groups contains group identifiers for any group that this user is
```

Comment:
Could also bump the cap ver so that we don't send it down to clients who don't understand it

Reply:
done